action-server/tests/actions/test_test_navigation_form.py

```diff
@@ -138,6 +138,15 @@
 }
 
 
+CLEARED_SLOTS = [
```

Comment:
Why not exporting this in `action-server/covidflow/actions/test_navigation_form.py`?

Reply:
Because I didn't think about it?

Reply:
ok :stuck_out_tongue: